package.json

```diff
@@ -59,14 +67,18 @@
   "devDependencies": {
     "babel-cli": "^6.18.0",
     "babel-core": "^6.21.0",
+    "babel-eslint": "^7.1.1",
```

Comment:
The default parser for standardjs/eslint can't read the flow annotations, you have to switch to the babel parser for that.